tests/tensor/rewriting/test_elemwise.py

```diff
@@ -163,6 +163,17 @@ def test_dimshuffle_on_broadcastable(self):
         # Check stacktrace was copied over correctly after rewrite was applied
         assert hasattr(g.outputs[0].tag, "trace")
 
+    def test_dimshuffle_lift_multi_out_elemwise(self):
+        x = float64("x")
+        outs = [x + 1, x + 2]
+        op = Elemwise(Composite([x], outs))
+
+        x = matrix("x")
+        outs = [out.T for out in op(x)]
+
+        g = FunctionGraph([x], outs)
+        assert not local_dimshuffle_lift.transform(g, g.outputs[0].owner)
```

Comment:
How does this test work?
I think it could use a few comments.

I tried:

```suggestion
        # Create an Op that returns x+1 and x+2
        x = float64("x")
        outs = [x + 1, x + 2]
        op = Elemwise(Composite([x], outs))

        # Create a graph that transposes the outputs of such results
        # to provoke a dimshuffle behind the scenes
        x = matrix("x")
        outs = [out.T for out in op(x)]
        g = FunctionGraph([x], outs)
        
        # Check that the dimshuffle lift rewrite doesn't apply to this graph
        # which tells us.....??????????
        assert not local_dimshuffle_lift.transform(g, g.outputs[0].owner)
```

Reply:
Added some comments WDYT?